test/throbber.js

```diff
@@ -26,14 +26,19 @@ module.exports = {
 		t.stderr.on("data", function (data) { err += data; });
 		t.on("exit", function () {
 			a.ok(out.length > 4, "Interval");
-			a(
-				startsWith.call(
-					out.join(""),
-					"START\x1b[31m-\x1b[39m\x1b[31m\b\\\x1b" +
-						"[39m\x1b[31m\b|\x1b[39m\x1b[31m\b/\x1b[39m\x1b[31m\b-\x1b[39m"
-				),
-				true, "Output"
-			);
+			var isNoColor = Boolean(process.env.NO_COLOR);
```

Comment:
I guess I'd follow same convention as in other files (first force color, then run same tests, then bring back _auto_)

Reply:
This part of the code is a special case, since it executes in a separate process, therefore it doesn't behave as expected. The best idea I could come up with to handle that was to simply rely on the state of the NO_COLOR variable on the system running the test. It's not ideal but I'm not sure how to support testing this unless you have an idea.

Reply:
Indeed, I forgot about that, in that case, this approach makes perfect sense